docs/src/layouts/HeroContainer.tsx

```diff
@@ -47,7 +47,7 @@ export default function HeroContainer({
 
   const renderRightWrapper = (sx?: BoxProps['sx']) => (
     <Box
-      id="hero-container-right-area"
+      ref={frame}
```

Comment:
Open https://mui.com/, you can <kbd>Tab</kbd> between the different elements in the hero.